File: Iniciante/1013/1013.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int a,b,c, maior;
    scanf("%d %d %d",&a,&b,&c);
    
    maior = (a+b+abs(a-b))/2;
    maior = (maior+c+abs(maior-c))/2;

    printf("%d eh o maior\n",maior);

    return 0;

}

File: Rust/Iniciante/1013.rs
use std::io;
use std::error::Error;

fn maior(a: i64, b: i64) -> i64 {
	return (a + b + (a - b).abs()) / 2;
}

fn main() -> Result<(), Box<dyn Error>> {
	let stdin = io::stdin();
	let mut buffer = String::new();

	stdin.read_line(&mut buffer)?;
	let mut tokens = buffer.split_whitespace();

	let a: i64 = tokens.next().unwrap().parse()?;
	let b: i64 = tokens.next().unwrap().parse()?;
	let c: i64 = tokens.next().unwrap().parse()?;

	let m: i64 = maior(a, maior(b, c));

	println!("{} eh o maior", m);

	Ok(())
}